app/src/main/java/org/fossasia/openevent/general/similarEvents/EventTopic.kt

```diff
@@ -0,0 +1,13 @@
+package org.fossasia.openevent.general.similarEvents
+
+import com.github.jasminb.jsonapi.LongIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("event-topic")
+data class EventTopic(
+        @Id(LongIdHandler::class)
+        val id: Long? = null,
+        val name: String? = null,
+        val slug: String? = null
```

Comment:
Add a foreign key to event

Reply:
Actually EventTopic is not being inserted into the database, so I thought that foriegn key is not needed

Reply:
Okay! Got it wrong :+1:

Reply:
You have to insert it into database as well

Reply:
Updated PR :+1: